SQLite stopped working

**Describe the bug**
I upgraded from a previous  version (I don't remember which one exactly) and SQLite doesn't work anymore.

```
2022-09-07T00:25:24.736313+01:00 [WARN ] with sqlite, only a single connection is supported
2022-09-07T00:25:24.7368331+01:00 [ERROR] Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
2022-09-07T00:25:24.7368331+01:00 [ERROR] could not open database
panic: could not open database

goroutine 1 [running]:
github.com/emvi/logbuch.(*StandardFormatter).Pnc(0xc0000da000?, {0xde9718?, 0xde9718?}, {0x0?, 0x0?, 0x0?})
        /home/runner/go/pkg/mod/github.com/emvi/logbuch@v1.2.0/standard_formatter.go:57 +0x3f
github.com/emvi/logbuch.(*Logger).Fatal(0xc0000da000, {0xde9718, 0x17}, {0x0, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/emvi/logbuch@v1.2.0/logger.go:132 +0x77
github.com/emvi/logbuch.Fatal(...)
        /home/runner/go/pkg/mod/github.com/emvi/logbuch@v1.2.0/funcs.go:55
main.main()
        /home/runner/work/wakapi/wakapi/main.go:126 +0x379
```

**System information**
Please provide information on:
* Wakapi version 2.3.8 (amd64)
* Operating system Windows 11
* Database (SQLite, MySQL, ... ?) SQLite

@muety Can you please delete all the arm64 releases?

```
$ file wakapi
wakapi: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=rt5oL3Dg5LrG_4H9LbSO/CxS-DGdQvYTZEM2fJCLz/V5xQ_5TmTdAMoYrAvcJP/nGHL8WvqdH8dWuQO0NE2, with debug_info, not stripped

$ ./wakapi
2022-09-07T07:45:50.005705269Z [WARN ] with sqlite, only a single connection is supported
2022-09-07T07:45:50.008835599Z [ERROR] Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
2022-09-07T07:45:50.008855279Z [ERROR] could not open database
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -64,8 +64,7 @@ jobs:
       shell: bash
       run: |
         GOOS=${{ matrix.GOOS }} GOARCH=${{ matrix.GOARCH }} \
-          CC=${{ matrix.CC }} CXX=${{ matrix.CXX }} CGO_ENABLED=1 \
-          go build -v ../
+          CC=${{ matrix.CC }} CGO_ENABLED=1 go build -v ../
 
     - name: Compress working folder (Windows PowerShell)
       working-directory: ./dist

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -64,7 +64,7 @@ jobs:
       shell: bash
       run: |
         GOOS=${{ matrix.GOOS }} GOARCH=${{ matrix.GOARCH }} \
-          CC=${{ matrix.CC }} CGO_ENABLED=1 go build -v ../
+          CC=${{ matrix.CC }} CGO_ENABLED=1 go build -v -ldflags '-w -s' ../
 
     - name: Compress working folder (Windows PowerShell)
       working-directory: ./dist

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -1,19 +1,37 @@
 name: Release
 
 on:
-  release:
-    types:
-      - published
+  push:
+  # release:
+  #   types:
+  #     - published
 
 jobs:
   release:
     name: 'Build, package and release to GitHub'
-    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        GOOS: [darwin, windows, linux]
-        GOARCH: [amd64, arm64]
+        include:
+          - platform: ubuntu-latest
+            GOOS: linux
+            GOARCH: amd64
+            CC: gcc
+          - platform: ubuntu-latest
+            package: gcc-aarch64-linux-gnu
+            GOOS: linux
+            GOARCH: arm64
+            CC: aarch64-linux-gnu-gcc
+          - platform: windows-latest
+            GOOS: windows
+            GOARCH: amd64
+          - platform: macos-latest
+            GOOS: darwin
+            GOARCH: amd64
+          - platform: macos-latest
+            GOOS: darwin
+            GOARCH: arm64
+    runs-on: ${{ matrix.platform }}
 
     steps:
 
@@ -25,6 +43,10 @@ jobs:
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
+
+    - name: Install gcc
+      if: "${{ matrix.package != '' }}"
+      run: sudo apt-get update && sudo apt-get -y install ${{ matrix.package }}
 
     - name: Set version
       shell: bash
@@ -40,15 +62,25 @@ jobs:
 
     - name: Build
       working-directory: ./dist
+      shell: bash
       run: |
-        GOOS=${{ matrix.GOOS }} GOARCH=${{ matrix.GOARCH }} go build -v ../
+        GOOS=${{ matrix.GOOS }} GOARCH=${{ matrix.GOARCH }} \
+          CC=${{ matrix.CC }} CXX=${{ matrix.CXX }} CGO_ENABLED=1 \
+          go build -v ../
+
+    - name: Compress working folder (Windows PowerShell)
+      working-directory: ./dist
+      if: "${{ matrix.GOOS == 'windows' }}"
+      run: |
+        Compress-Archive -Path .\wakapi.exe, .\config.yml -DestinationPath wakapi_${{ matrix.GOOS }}_${{ matrix.GOARCH }}.zip
 
     - name: Compress working folder
       working-directory: ./dist
+      if: "${{ matrix.GOOS != 'windows' }}"
       run: |
         zip -9 wakapi_${{ matrix.GOOS }}_${{ matrix.GOARCH }}.zip *
 
-    - name: Upload built executable to Release
-      uses: softprops/action-gh-release@v1
-      with:
-        files: ./dist/*.zip
+    # - name: Upload built executable to Release
+    #   uses: softprops/action-gh-release@v1
+    #   with:
+    #     files: ./dist/*.zip

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -1,10 +1,9 @@
 name: Release
 
 on:
-  push:
-  # release:
-  #   types:
-  #     - published
+  release:
+    types:
+      - published
 
 jobs:
   release:
@@ -80,7 +79,7 @@ jobs:
       run: |
         zip -9 wakapi_${{ matrix.GOOS }}_${{ matrix.GOARCH }}.zip *
 
-    # - name: Upload built executable to Release
-    #   uses: softprops/action-gh-release@v1
-    #   with:
-    #     files: ./dist/*.zip
+    - name: Upload built executable to Release
+      uses: softprops/action-gh-release@v1
+      with:
+        files: ./dist/*.zip

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -12,11 +12,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - platform: ubuntu-latest
+          - platform: ubuntu-18.04
             GOOS: linux
             GOARCH: amd64
             CC: gcc
-          - platform: ubuntu-latest
+          - platform: ubuntu-18.04
             package: gcc-aarch64-linux-gnu
             GOOS: linux
             GOARCH: arm64

